src/lib/y2storage/storage_env.rb

```diff
@@ -89,6 +92,15 @@ def luks2_available?
       active?(ENV_LUKS2_AVAILABLE, default: false)
     end
 
+    # Whether YaST should reuse existing LVM
+    #
+    # see jsc#PED-6407 or jsc#IBM-1315
+    #
+    # @return [Boolean, nil] boolean as explicitly set by user, nil if user set nothing
+    def requested_lvm_reuse
+      active?(ENV_REUSE_LVM, default: nil)
```

Comment:
Sorry to be picky about this, but the problem already mentioned about boolean and nil is here just moved one level down. :-)

Still, there is a method `active?` that returns something that is not a boolean. You are actually deviating from the original meaning of that method which explicitly declares both the return value and the parameter `default` to be booleans.

If you want to introduce support for nil values, please do it at `requested_lvm_reuse`, but don't twist `active?`.

Reply:
ok, i tried and it didn't pass ... it happens ;-)

So another try is here.